.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
```

Comment:
We use semicolons through-out the project. Why did you choose to remove all of them? Otherwise nice config.

Reply:
I gotta admit I didn't spend too much time thinking about specific style choices - i actually should have created a draft PR instead.
Anyhow, I'll adjust this soon. Also the README part of course. Thanks for your feedback!